src/components/Home/Keshav/Week5/Counter/index.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { CounterWrapper } from './styles';
+import { useSelector, useDispatch } from 'react-redux';
+import { incrementCounter, decrementCounter } from '../redux/actions/CounterActions';
+import { RootState } from '../redux/actions/types';
+
+const Counter: React.FC = () => {
+  const count = useSelector((state: RootState) => state.count);
+  const dispatch = useDispatch();
+
+  const increment = (): void => {
+    dispatch(incrementCounter());
+  };
+
+  const decrement = (): void => {
+    dispatch(decrementCounter());
+  };
+
+  return (
+    <CounterWrapper>
+      <h1>Counter using Redux</h1>
+      <span className='counterHeading'>Counter : {count}</span>
+      <br />
```

Comment:
please use css if you want to add extra lines in between elements. you can use flex to display inline elements in blocks.
break tag is commonly used where you want to create a line break within a paragraph or text.